roles/gpu_operator_deploy_from_operatorhub/tasks/deploy_clusterpolicy.yml

```diff
@@ -29,24 +29,11 @@
     retries: 15
     delay: 30
 
-- name: Get the clusterpolicy of the GPU Operator from OperatorHub CSV
-  shell:
-    set -o pipefail;
-    oc get "{{ gpu_operator_csv_name_cmd.stdout }}"
-       -n openshift-operators
-       -ojson
-    | jq -r '.metadata.annotations."alm-examples"'
-    | jq .[0] > "{{ artifact_extra_logs_dir }}/gpu_operator_clusterpolicy.json"
-  register: operatorhub_clusterpolicy
-  until: operatorhub_clusterpolicy.rc == 0
-  retries: 20
-  delay: 15
-
 - block:
-  - name: Create the clusterPolicy CR for the GPU Operator
-    command: oc apply -f "{{ artifact_extra_logs_dir }}/gpu_operator_clusterpolicy.json"
-    register: test_clusterpolicy_cr
-    until: test_clusterpolicy_cr.rc != 1
+  - name: Wait for the GPU Operator ClusterPolicy CRD to appear
```

Comment:
Can you explain the reason for this change in the commit message?

Reply:
```
    gpu_operator_deploy_from_operatorhub: wait for CRD instead of waiting for CR creating

    This commit improves the separation between the initial deployment of
    the operator (done by OLM) and the following phase, where the
    ClusterPolicy CR is instantiated.

    We now explicitly wait for the crd/clusterpolicy.nvidia.com to
    appear (OLM deployment),

    and then instantiate the ClusterPolicy CR (beginning of the operands
    deployment).
```